src/components/Input/Input.module.scss

```diff
@@ -0,0 +1,10 @@
+.input {
+  width: 100%;
+  height: 38px;
+  margin-bottom: 6px;
```

Comment:
The component doesn't need default margins.

Reply:
Fixed it